gwpy/io/datafind.py

```diff
@@ -40,14 +40,13 @@
 import warnings
 from functools import wraps
 from http.client import HTTPException
-from urllib.parse import urlparse
 
 from ligo.segments import (
     segment as LigoSegment,
     segmentlist as LigoSegmentList,
 )
 from ..time import to_gps
-from .cache import (cache_segments, read_cache_entry, _iter_cache)
+from .cache import (cache_segments, read_cache_entry, _iter_cache, file_path)
```

Comment:
Just a minor question, should these imports come from `.utils`?

Reply:
Yes, I have made that change with an amended commit, thanks.